.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -100,48 +106,48 @@ jobs:
       - generate_bom
     runs-on: ubuntu-latest
     steps:
-      - name: Download artifact       
+      - name: Download artifact
         uses: actions/download-artifact@v4
         with:
-          name: ${{ github.run_number }}-sbom.json
+          name: ${{ github.run_number }}${{ inputs.package-directory }}.sbom.json
       - name: Test SBOM (HIGH,CRITICAL)
         uses: aquasecurity/trivy-action@master
         with:
           exit-code: '0' # Don't kill, this run is just output
           severity: 'HIGH,CRITICAL'
           ignore-unfixed: true
-          scan-ref: ${{ env.BOM_FILE_PATH }} # Need; See /aquasecurity/trivy-action/pull/314
+          scan-ref: ${{ inputs.working-directory }}/${{ env.BOM_FILE_NAME }} # Need; See /aquasecurity/trivy-action/pull/314
           scan-type: 'sbom'
           scanners: 'vuln'
           format: 'table'
           output: trivy-sbom-HIGH-CRITICAL-vuln-report.txt
       - run: cat trivy-sbom-HIGH-CRITICAL-vuln-report.txt
-      - name: Download artifact       
+      - name: Download artifact
```

Comment:
How is this download any different from line 103? (Same file, same runner environment)

Reply:
(Is the file overwritten?)

Reply:
Copy paste mistake; Fixed. [462968d](https://github.com/telicent-oss/shared-workflows/pull/45/commits/462968d0a2dd86fafe090c024d5f223022887ac4)